vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SlideManagerTest.java

```diff
@@ -0,0 +1,159 @@
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        List<Sequence> actualSequenceSlideListNotPresent = slideManagerToTest.getSlideSequences(slideIdNotInSequence,
+                moduleId);
+        Assert.assertTrue(actualSequenceSlideListNotPresent.isEmpty());
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdPresent() throws SlideDoesNotExistException {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        Slide slide = new Slide();
+        slide.setId(slideId);
+        Optional<Slide> slideObj = Optional.of(slide);
+        Mockito.when(slideRepo.findById(slide.getId())).thenReturn(slideObj);
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.verify(slideRepo).delete((Slide) slide);
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdNotPresent() throws SlideDoesNotExistException {
+
+        Mockito.when(slideRepo.findById(slideIdNotPresent)).thenReturn(Optional.empty());
+        slideManagerToTest.deleteSlideById(slideIdNotPresent, moduleId);
+        Mockito.verify(slideRepo, Mockito.never()).deleteById(slideIdNotPresent);
+
+    }
+    
+    @Test
+    public void test_deleteSlideById_slideIdIsNull() {
+
+        Mockito.when(slideRepo.findById(null)).thenReturn(Optional.empty());
+        slideManagerToTest.deleteSlideById(null, moduleId);
+        Mockito.verify(slideRepo, Mockito.never()).deleteById(slideIdNotPresent);
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdPresentManySequences() throws SlideDoesNotExistException {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+
+        sequenceObjOther.setId(sequenceIdOther);
+        sequenceObjOther.setSlides(slidesList);
+        sequenceObjOther.setSlides(slidesListOther);
+
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        sequencesList.add(sequenceObjOther);
+        String slideIdBeforeDeletion = sequencesList.get(1).getSlides().get(0).getId();
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        ISlide slideObj = slidesList.get(0);
+        Mockito.when(slideRepo.findById(slideObj.getId())).thenReturn(Optional.of((Slide) slideObj));
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.verify(slideRepo).delete((Slide) slideObj);
+        String slideIdAfterDeletion = sequencesList.get(1).getSlides().get(0).getId();
+
+        Assert.assertNotEquals(slideIdBeforeDeletion, slideIdAfterDeletion);

```

Comment:
if the method changed the order of the objects, this would pass although the method has a bug. why don't you test if the the object is still in the list?